libs/librrgraph/src/base/rr_graph_reader.cpp

```diff
@@ -21,8 +21,6 @@
 #include <fstream>
```

Comment:
Suggest to move all the reader and writer source files to ``libs/librrgraph/src/io``